test/functional/http/get/node.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var node = require('../../../node.js');
+var http = require('../../../common/httpCommunication.js');
+
+describe('GET /api/node/constants', function () {
+
+	var constantsResponse;
+
+	before(function (done) {
+		http.get('/api/node/constants', function (err, res) {
+			constantsResponse = res.body;
+			done();
+		});
+	});
+
+	it('should return a result containing build = ""', function () {
+		node.expect(constantsResponse).to.have.property('build').to.equal('');
+	});
+
+	it('should return a result containing commit as a string of length 40', function () {
+		node.expect(constantsResponse).to.have.property('commit').that.is.a('string').and.has.a.lengthOf(40);
+	});
+
+	it('should return a result containing epoch as a string date', function () {
+		node.expect(constantsResponse).to.have.property('epoch').that.is.a('string');
+		node.expect(new Date(constantsResponse.epoch)).not.to.equal('Invalid Date');
+	});
+
+	it('should return a result containing nethash = "198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d"', function () {
+		node.expect(constantsResponse).to.have.property('nethash').equal('198f2b61a8eb95fbeed58b8216780b68f697f26b849acf00c8c93bb9b24f783d');
```

Comment:
Will the nethash always be this, or will it depend on the order of running this test?

Reply:
nethash will remain the same always - devnet